cluster-operator/src/test/java/io/strimzi/operator/cluster/ResourceUtils.java

```diff
@@ -573,6 +576,21 @@ public static Admin adminClient()   {
         }
         when(mock.describeMetadataQuorum()).thenReturn(dmqr);
 
+        DescribeClientQuotasResult dcqr;
+        try {
+            Constructor<DescribeClientQuotasResult> declaredConstructor = DescribeClientQuotasResult.class.getDeclaredConstructor(KafkaFuture.class);
+
+            final ClientQuotaEntity defaultUserEntity = new ClientQuotaEntity(Collections.singletonMap(ClientQuotaEntity.USER, null));
```

Comment:
Map.of() instead of singleton map? Also, maybe you can just inline it?

Reply:
`Map.of()` throws error (NPE) when you pass there null value and I need here the `null` value to pick the default user there.